<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="2" tests="12" name="SIP.demoCustomer_Query" time="45.098" errors="0" timestamp="2025-02-21T17:05:20 IST" skipped="0">
  <testcase classname="SIP.demoCustomer_Query" name="Dealer_User_Login1" time="6.405"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Click_On_SIP" time="15.911">
    <failure message="Test Failed: Wrong Booking list Url" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test Failed: Wrong Booking list Url
at org.testng.Assert.fail(Assert.java:99)
at SIP.Dealer_User_Login.Click_On_SIP(Dealer_User_Login.java:83)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Click_On_SIP -->
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Click_On_Additional_Fillter_To_Clear_Data" time="5.747"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Click_On_Customer_Page" time="0.847"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Customer_Query" time="10.761">
    <failure message="(In-Pending Booking),Tentative Waiting period not visible as Expected Time. " type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: (In-Pending Booking),Tentative Waiting period not visible as Expected Time. 
at org.testng.Assert.fail(Assert.java:99)
at SIP.demoCustomer_Query.Click_On_Customer_Query(demoCustomer_Query.java:139)
at SIP.demoCustomer_Query.Customer_Query(demoCustomer_Query.java:80)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Customer_Query -->
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Select_PRODUCT" time="0.763"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Write_Query_And_SAVE" time="0.836"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Click_On_Query_History" time="1.511"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Customer_Remarks" time="0.079"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="CRE_Remarks" time="1.622"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Status_Closed" time="0.097"/>
  <system-out/>
  <testcase classname="SIP.demoCustomer_Query" name="Match_status_For_Product" time="0.519"/>
  <system-out/>
</testsuite> <!-- SIP.demoCustomer_Query -->
